### DISCUSSION TITLE
Duplicate Race Control (RCM) sound notifications

### DISCUSSION THREAD

**Discussion by raidenelite** [f1multiviewer/issue-tracker#207] [Feature requests] [answered]
Hello, during today qualifications i had an annoying issue with sound notifications for the RCM when having the pop out window playing the sound two times.

System: Windows 10

**Comment by LapsTimeOFF** [comment]
Also present on : MacOS

**Comment by f1multiviewer** [comment, marked as answer]
This is "solved" now, as you can turn off RCMs on the main timing screen if you pop them out, in settings.
